lib/dotcom/application.ex

```diff
@@ -27,11 +27,12 @@ defmodule Dotcom.Application do
     children =
       [
         {Application.get_env(:dotcom, :cache, Dotcom.Cache.Multilevel), []},
-        {Dotcom.RateLimit, [clean_period: 60_000 * 10]}
+        {Dotcom.RateLimit, [clean_period: 60_000 * 10]},
+        {Finch, name: Dotcom.Finch, pools: %{default: [size: 300, count: 10]}}
```

Comment:
Silly question: Do we want this for the `:test` environment too, or just deployed envs?